src/ape/contracts/base.py

```diff
@@ -149,11 +151,52 @@ def __repr__(self) -> str:
         # `<ContractName 0x1234...AbCd>.method_name`
         return f"{self.contract.__repr__()}.{self.abis[-1].name}"
 
+    @log_instead_of_fail()
+    def _repr_pretty_(self, printer, cycle):
+        """
+        Show the NatSpec of a Method in any IPython console (including ``ape console``).
+        """
+        if info := self.info:
+            text = Text(info)
+            rich_print(f"<natspec>\n{info}")
```

Comment:
Whats the `<natspec>` tag do? Or is this just debugging for now?

Reply:
I am trying to think of a way to get all the info starting on a newline after `Out[1]:` part or else the indentation seems weird on the first line